tests/base_test.py

```diff
@@ -135,16 +107,77 @@ def setUpClass(cls):
+        cls.loadOntology()
+        if not cls.graph_fixtures:
+            return
+        for path in test_settings.RESOURCE_GRAPH_LOCATIONS:
+            file_paths = [
+                file_path
+                for file_path in os.listdir(path)
+                if file_path.endswith(".json")
+                and Path(file_path).stem in cls.graph_fixtures
+            ]
+            for file_path in file_paths:
+                with captured_stdout():
+                    with open(os.path.join(path, file_path), "r") as f:
+                        archesfile = JSONDeserializer().deserialize(f)
+                        errs, importer = ResourceGraphImporter(
+                            archesfile["graph"], overwrite_graphs=True
+                        )
+
+    @classmethod
+    def legacy_load_testing_package(cls):
+        """Do not write new tests with this method."""
+        with (
+            captured_stdout(),
+            mock.patch(
+                "arches.management.commands.packages.Command.update_resource_geojson_geometries"
+            ),
+        ):
+            management.call_command(
+                "packages",
+                "-o import_reference_data -s tests/fixtures/testing_prj/testing_prj/pkg/reference_data/concepts/Test-scheme.xml".split(),
+            )
+            management.call_command(
+                "packages",
+                [
+                    "-o",
+                    "import_reference_data",
+                    "-s",
+                    "tests/fixtures/testing_prj/testing_prj/pkg/reference_data/concepts/4.3 Test RDM Thesaurus.xml",
+                ],
+            )
+            management.call_command(
+                "packages",
+                "-o import_reference_data -s tests/fixtures/testing_prj/testing_prj/pkg/reference_data/collections/Test-scheme-collections.xml".split(),
+            )
+            management.call_command(
+                "packages",
+                [
+                    "-o",
+                    "import_reference_data",
+                    "-s",
+                    "tests/fixtures/testing_prj/testing_prj/pkg/reference_data/collections/4.3 Test RDM Collections.xml",
+                ],
+            )
+            management.call_command(
+                "packages",
+                "-o load_package -s tests/fixtures/testing_prj/testing_prj/pkg -ow True -y".split(),
+            )
+
+        path_to_cheesy_image = Path(settings.MEDIA_ROOT) / "uploadedfiles" / "test.png"
+        cls.addClassCleanup(os.unlink, path_to_cheesy_image)
 
     @classmethod
     def tearDownClass(cls):
-        cursor = connection.cursor()
         sql = "DELETE FROM public.oauth2_provider_application WHERE id = 44;"
-        cursor.execute(sql)
+        with connection.cursor() as cursor:
+            cursor.execute(sql)
         super().tearDownClass()
-
-    @classmethod
-    def deleteGraph(cls, root):
-        graph = Graph.objects.get(graphid=str(root))
```

Comment:
Now that we're being more careful about syncing ES during tests that care about ES, there's less of a need to run proxy model deletes to ensure side effects are performed. We can just rely on the transaction rolling back to leave PG in the correct state.